dnf/cli/output.py

```diff
@@ -1666,8 +1680,8 @@ def _simple_pkg(pkg, prefix_len, was_installed=False, highlight=False,
             else:
                 print(_("Command Line   :"), old.cmdline)
 
-        if not isinstance(old.tid, list):
-            addon_info = self.history.return_addon_data(old.tid)
+        if len(tids) == 1:
```

Comment:
old.tid is always a list - C is not as dynamic as Python. It's used for differentiation between single and merged transaction. We don't print that data for merged transactions...